pkg/media/pcmlocaltrack.go

```diff
@@ -172,6 +165,10 @@ func (t *PCMLocalTrack) WriteSample(chunk media.PCM16Sample) error {
 		return errors.New("track is closed")
 	}
 
+	if t.muted.Load() {
+		return errors.New("track is muted")
+	}
```

Comment:
Can we not return an error here? Users could log error if they get an error return. With remote mute, users could still be writing samples and getting an error back. I think it is fine to document that `WriteSample` will drop samples when muted (local or remote) and not return an error here.

Reply:
I'm not too sure about this, they should not be writing samples on mute, so an error makes sense. 

As a user, if I don't have the code to handle remote mute, the error will help me spot it.

Reply:
That means application has to maintain state and not write or filter for mute errors. Yeah, not handling remote mute would be more obvious, but once an app fixes that, it would need to maintain mute state at app level also or filter errors. Don't have strong opinions, but looking a lot of code for specific error handling or maintaining state in two places, wanted to suggest that.